### PR TITLE
Opening the html reports in browse instead of VSCode #354

### PR DESCRIPTION
**Pull request by Apoorva-GA**

### CODE REVIEW

File: src/execution/gaugeExecutor.ts [hunk 1]
```diff
@@ -315,10 +311,22 @@ export class GaugeExecutor extends Disposable {
...
+            }),
+            commands.registerCommand(GaugeVSCodeCommands.ShowReport, () => {
+                this.openReport();
             })
         ));
     }
 
+    private openReport() {
+        let projectRoot = ProjectFactory.get(this.gaugeWorkspace.getDefaultFolder()).root();
+        let url = join(projectRoot, "/reports/html-report/index.html");
```
Comment (negiDharmendra) [inline src/execution/gaugeExecutor.ts]: `reports` directory is configurable through `gauge_reports_dir` property. If the user has configured it to something else then they won't be able to open it through vscode command palette.  The same issue will happen when `overwrite_reports` is set to `false`.

Comment (Apoorva-GA) [reply src/execution/gaugeExecutor.ts]: yes, we need to handle these cases. Is it ok if separate cards are raised for the same and looked into?

Comment (negiDharmendra) [reply src/execution/gaugeExecutor.ts]: Please create a separate issue for the same.

Comment (Apoorva-GA) [reply src/execution/gaugeExecutor.ts]: Done. https://github.com/getgauge/gauge-vscode/issues/362

**Pull request merged**